app/src/main/res/layout/activity_create.xml

```diff
@@ -136,26 +135,42 @@
 
     <Button
         android:id="@+id/signup_create"
-        android:layout_width="wrap_content"
+        android:layout_width="0dp"
```

Comment:
just wondering though: is this supposed to be 0dp?

Reply:
I think so. I made it so the two buttons are same width, I think android does something automatically.